xbmc/GUIInfoManager.cpp

```diff
@@ -7730,6 +7749,14 @@ const infomap playlist[] =       {{ "length",           PLAYLIST_LENGTH },
 ///     @return The backend number.
 ///     <p>
 ///   }
+///   \table_row3{   <b>`PVR.BackendInstanceName`</b>,
+///                  \anchor PVR_BackendInstanceName
+///                  _string_,
+///     @return The name used by the PVR client addon instance.
```

Comment:
You should be clear here that the value returned here is for the currently playing list item.

Reply:
It's not only playing items.  It applies to EPG, channels, recordings, etc, as well.  Do you still want that change made?

Reply:
Context for "PVR.*" is playing item. As you have not yet implemented this label, you will see if you do.

Reply:
https://github.com/xbmc/xbmc/blob/77eacb8b9c665b7b5db0efa8892078213efec163/addons/skin.estuary/xml/Variables.xml#L801-L803
https://github.com/xbmc/xbmc/blob/77eacb8b9c665b7b5db0efa8892078213efec163/addons/skin.estuary/xml/Includes_PVR.xml#L443

Have a look at the screenshots above.  For example:
![image](https://github.com/xbmc/xbmc/assets/127641886/409d0712-0d16-45d9-acc4-428cda07536c)

The label returns a value in the EPG screen, the recordings screen and the timers screen.

Do we have a misunderstanding over the term 'playing item'?  I think that a 'playing item' is a PVR recording currently being watched.

Reply:
> Do we have a misunderstanding over the term 'playing item'? I think that a 'playing item' is a PVR recording currently being watched.

 That understanding is generally okay. But for "Listitem" you need a context with a list containing items, obviously (like recordings window, timers window).
 
 This is not the case for example when you just pull up the OSD while playing a recording full screen. There is no item list in this cntext, just the playing item you can obtain from the system.

Reply:
`PVR.BackendInstanceName` has been removed from the PR.  (See note following)

Reply:
Okay, if we need it we can add it at any point in time later.